docs/pages/styled-jsx/+Page.mdx

```diff
@@ -1,13 +1,20 @@
 import { Example, UiFrameworkExtension, CommunityNote } from '../../components'
 import { Link } from '@brillout/docpress'
 
-<CommunityNote tool="styled-jsx" url="https://github.com/vercel/styled-jsx" hasExtension={false} />
+<CommunityNote tool="styled-jsx" url="https://github.com/vercel/styled-jsx" hasExtension="react" />
 
-Example:
+## Manual integration
+
+If you use <Link href="/vike-react">`vike-react`</Link>, you can manually integrate styled-jsx by using:
+- <Link href="/onBeforeRenderHtml" doNotInferSectionTitle />
+- <Link href="/Wrapper" doNotInferSectionTitle />
+- <Link href="/onAfterRenderHtml" doNotInferSectionTitle />
+
+Example of using Vike with styled-jsx:
  - <Example timestamp="2023.07" repo="jeremypress/vite-ssr-styled-jsx" />
    > vite-plugin-ssr was the [previous name of Vike](https://vite-plugin-ssr.com/vike).
 
-To use Vike with `styled-jsx`:
+To use Vike with `styled-jsx` without <Link href="/vike-react">`vike-react`</Link>:
```

Comment:
:100: Good catch. Intervention-=2.

Let's create a new section for it ``## Without `vike-{react,vue,solid}``` (it's a common convention, see for example https://vike.dev/ssr#without-vike-react-vue-solid).